docs/get-started/backup/restore_MongoDB_backups.md

```diff
@@ -0,0 +1,126 @@
+# Restore a MongoDB backup
+
+## Restore compatibility
+
+MongoDB backups can only be restored to the same service they were created from.
+
+To restore a backup:
+
+1. Go to <i class="uil uil-history"></i> **Backup > All backups** and find the backup that you want to restore.
+2. Click the arrow in the **Actions** column to check all the information for the backup, then click ![](../_images/dots-three-vertical.png) **> Restore from backup**.
+3. In the **Restore from backup** dialog, select **Same service** to restore to a service with identical properties.
```

Comment:
User cannot select it for mongo, it's already preselected and cannot be changed.